Source/Initialization/PlasmaInjector.cpp

```diff
@@ -160,12 +160,12 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
         pp.getarr("multiple_particles_vel_z", multiple_particles_vel_z);
         pp.getarr("multiple_particles_weight", multiple_particles_weight);
         AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
-            (multiple_particles_pos_x.size() == multiple_particles_pos_y.size() &
-             multiple_particles_pos_x.size() == multiple_particles_pos_z.size() &
-             multiple_particles_pos_x.size() == multiple_particles_vel_x.size() &
-             multiple_particles_pos_x.size() == multiple_particles_vel_y.size() &
-             multiple_particles_pos_x.size() == multiple_particles_vel_z.size() &
-             multiple_particles_pos_x.size() == multiple_particles_weight.size()),
+            ((multiple_particles_pos_x.size() == multiple_particles_pos_y.size()) &
+             (multiple_particles_pos_x.size() == multiple_particles_pos_z.size()) &
+             (multiple_particles_pos_x.size() == multiple_particles_vel_x.size()) &
+             (multiple_particles_pos_x.size() == multiple_particles_vel_y.size()) &
+             (multiple_particles_pos_x.size() == multiple_particles_vel_z.size()) &
+             (multiple_particles_pos_x.size() == multiple_particles_weight.size())),
```

Comment:
Those `&` should be `&&`
```suggestion
            ((multiple_particles_pos_x.size() == multiple_particles_pos_y.size()) &&
             (multiple_particles_pos_x.size() == multiple_particles_pos_z.size()) &&
             (multiple_particles_pos_x.size() == multiple_particles_vel_x.size()) &&
             (multiple_particles_pos_x.size() == multiple_particles_vel_y.size()) &&
             (multiple_particles_pos_x.size() == multiple_particles_vel_z.size()) &&
             (multiple_particles_pos_x.size() == multiple_particles_weight.size())),
```

Reply:
OK, thanks, I included this additional fix in https://github.com/ECP-WarpX/WarpX/pull/1747/commits/862d6f2ca2116d09a92b5b6be6c0908cc9fb8aaf.